frontend/src/components/samples/SampleTableColumns.tsx

```diff
@@ -103,6 +105,55 @@ export const SAMPLE_COLUMN_DEFINITIONS: { [key in SampleColumnID]: SampleColumn
 		},
 	},
 
+	[SampleColumnID.PARENT_CONTAINER]: {
+		columnID: SampleColumnID.PARENT_CONTAINER,
```

Comment:
I think we typically use 2 spaces for indentation on the frontend side to keep the indentation from getting too great.